src/main/resources/org/springframework/integration/aws/config/xml/spring-integration-aws-1.0.xsd

```diff
@@ -61,165 +75,175 @@
 		</xsd:attribute>
 	</xsd:attributeGroup>
 
-	<xsd:element name="s3-outbound-channel-adapter">
-		<xsd:complexType>
+	<xsd:complexType name="outboundS3BaseType">
+		<xsd:choice minOccurs="0" maxOccurs="2">
+			<xsd:element ref="integration:poller" minOccurs="0" maxOccurs="1"/>
+			<xsd:element name="request-handler-advice-chain" type="integration:handlerAdviceChainType" minOccurs="0" maxOccurs="1" />
```

Comment:
Well, I don't know right now already.
I'm trying here to convince you that for your case you don't need to wait for any release, but you still want to contribute and what is bad to that code which is supposed to be deprecated and removed. 
Rather then really code to the Framework :wink: